projects/adi_vna_8ch/zcu102/Makefile

```diff
@@ -0,0 +1,43 @@
+###############################################################################
+
```

Comment:
Same comments as for adi_vna_8ch_bd.tcl

Reply:
Should I skip this comment, since the file is autogenerated?